samples/ComputeSharp.SwapChain/Program.cs

```diff
@@ -35,28 +35,35 @@ class Program
 
     static void Main()
     {
-        Console.WriteLine("Available samples:");
-        Console.WriteLine();
+        int index;
 
-        for (int i = 0; i < Samples.Length; i++)
+        do
         {
-            Console.WriteLine($"{i}: {Samples[i].GetType().GenericTypeArguments[0].Name}");
-        }
+            Console.WriteLine("Available samples:");
+            Console.WriteLine();
 
-        Console.WriteLine();
+            for (int i = 0; i < Samples.Length; i++)
+            {
+                Console.WriteLine($"{i}: {Samples[i].GetType().GenericTypeArguments[0].Name}");
+            }
 
-        int index;
+            Console.WriteLine($"{Samples.Length}+: Exit (Use Escape, 'Q', or Alt+F4 to exit a sample once chosen.)");
```

Comment:
Nit: missing spacing and extra dot.
```suggestion
            Console.WriteLine($"{Samples.Length}+: Exit (Use Escape, 'Q', or Alt + F4 to exit a sample once chosen)");
```